src/nrtk_explorer/library/nrtk_transforms.py

```diff
@@ -12,7 +13,13 @@
 logger.setLevel(logging.INFO)
 
 try:
-    from pybsm.otf import darkCurrentFromDensity
+    import pybsm
+
+    if Version(pybsm.__version__) < Version("0.5"):
+        import pybsm.otf.darkCurrentFromDensity as dark_current_from_density
+    else:
+        import pybsm.otf.dark_current_from_density as dark_current_from_density
```

Comment:
Should we litter our codebase with these conditionals as pybsm develops?  Does anyone care if we support the old pybsm versions?